crates/red_knot_python_semantic/resources/mdtest/subscript/bytes.md

```diff
@@ -23,6 +23,18 @@ y = b[-6]  # error: [index-out-of-bounds] "Index -6 is out of bounds for bytes l
 reveal_type(y)  # revealed: Unknown
 ```
 
+## Slices
+
+```py
+b = b"\x00abc\xff"
+
+reveal_type(b[0:2])  # revealed: Literal[b"\x00a"]
+reveal_type(b[-3:])  # revealed: Literal[b"bc\xff"]
```

Comment:
I didn't bother repeating all tests here, since that seems sufficiently covered by the tuple/string cases + the slicing unit tests.